src/test/groovy/nebula/plugin/publishing/maven/ResolvedMavenPluginIntSpec.groovy

```diff
@@ -58,6 +58,37 @@ class ResolvedMavenPluginIntSpec extends IntegrationSpec {
         asmDep.version.text() == '2.2.3'
     }
 
+    def 'compatible with nebula.provided-base plugin'() {
+        writeHelloWorld('nebula.hello')
+        buildFile << '''
+            buildscript {
+                repositories { jcenter() }
+                dependencies { classpath "com.netflix.nebula:gradle-extra-configurations-plugin:2.2.0" }
```

Comment:
I'm glad you suggested this; interestingly, just adding a `provided` configuration (and having `compile` extend from it) isn't sufficient to reproduce the failure. The POM patching performed by the `nebula.provided-base` plugin appears to be what disrupts `ResolvedMavenPlugin`. A workaround for the issue is to apply the `nebula.provided-base` plugin *after* `nebula.nebula-maven-publishing`.